src/Frontend/Core.hs

```diff
@@ -642,8 +642,8 @@ cancelButton p = a_ [class_ "btn", href_ $ redirectOf p ()] "Abbrechen"
 -- the entire sequence of requests the client presents the same cookie, not only over those
 -- sub-sequences where the user is logged-in.)
 data Frame body
-    = Frame { _frameUser :: User, _frameBody :: body, _frameMessages :: [StatusMessage] }
-    | PublicFrame               { _frameBody :: body, _frameMessages :: [StatusMessage] }
+    = Frame { _frameUser :: User, _frameBody :: body, _frameMessages :: [StatusMessage], _frameDevMode :: Bool }
+    | PublicFrame               { _frameBody :: body, _frameMessages :: [StatusMessage], _frameDevMode :: Bool }
```

Comment:
It's looking more and more like the `_frameUser` should be a `Maybe` and there should be only one constructor, but let's not try that change here.